buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -2522,6 +2522,8 @@ class BeamModulePlugin implements Plugin<Project> {
           project.exec {
             environment "EXPANSION_JAR", expansionJar
             environment "EXPANSION_PORT", javaExpansionPort
+            //Use hardcoded kafka temporarily until hosted Kafka on Google is available. TODO: johnjcasey
+            environment "BOOTSTRAP_SERVER", 'theotherjohn-nokill-kafka-c-m:9092'
```

Comment:
do you have README to create this Kafka instance? FYI. I added mine under https://github.com/apache/beam/tree/master/.test-infra/pubsub.

Reply:
I created it with an internal script. In principle, any Kafka cluster hosted on GCP in apache beam testing will do.

This server also should only exist temporarily, as I plan to migrate to google managed kafka once that comes on line early 2024

Reply:
For README, note that we already have https://github.com/apache/beam/tree/master/.test-infra/kafka and one can use it to deploy kafka cluster, k8s based.

Reply:
honestly not comfortable add a private domain to BeamModulePlugin (even with sort of PII). Can we configure it some more specific path, like build.gradle or tox file that invokes this test?

Reply:
agreed. I'll rework this